cf/test/test_Field.py

```diff
@@ -2235,23 +2235,23 @@ def test_Field_del_domain_axis(self):
 
     def test_Field_percentile(self):
         f = cf.example_field(1)
-        for chunksize in self.chunk_sizes:
-            cf.chunksize(chunksize)
-            # Percentiles taken across *all axes*
-            ranks = [[30, 60, 90], [20], 80]  # include valid singular form
-
-            for rank in ranks:
-                # Note: in cf the default is squeeze=False, but numpy has an
-                # inverse parameter called keepdims which is by default False
-                # also, one must be set to the non-default for equivalents.
-                # So first cases (n1, n1) are both squeezed, (n2, n2) are not:
-                a1 = numpy.percentile(f, rank)  # has keepdims=False default
-                b1 = f.percentile(rank, squeeze=True)
-                self.assertTrue(b1.allclose(a1, rtol=1e-05, atol=1e-08))
-                a2 = numpy.percentile(f, rank, keepdims=True)
-                b2 = f.percentile(rank)  # has squeeze=False default
-                self.assertTrue(b2.shape, a2.shape)
-                self.assertTrue(b2.allclose(a2, rtol=1e-05, atol=1e-08))
+
+        # Percentiles taken across *all axes*
```

Comment:
This is making me think that it might be good to blitz all of the `for chunksize in self.chunk_sizes:` remaining in the test suite in one go:`git grep "for chunksize" | wc -l` says `6`, which isn't many now (most were in `test_Data` pre-migration, obviously) but enough to perhaps warrant a dedicated PR)?

Reply:
Sounds good to me - would you like to do it?

Reply:
Sure, I'll add it to my list (medium-to-low priority unless you say otherwise).